File: app/src/main/java/com/exory550/exorypad/android/ExorypadApplication.java
package com.exory550.exorypad.android;

import android.app.Application;
import com.exory550.exorypad.di.exorypadModule;
import org.koin.android.ext.koin.androidContext;
import org.koin.core.context.GlobalContext.startKoin;

public class ExorypadApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        startKoin {
            androidContext(this@ExorypadApplication);
            modules(exorypadModule);
        }
    }
}

File: app/src/main/java/com/exory550/exorypad/android/ExorypadApplication.kt
package com.exory550.exorypad.android

import android.app.Application
import com.exory550.exorypad.di.exorypadModule
import org.koin.android.ext.koin.androidContext
import org.koin.core.context.GlobalContext.startKoin

class ExorypadApplication: Application() {
    override fun onCreate() {
        super.onCreate()
        startKoin {
            androidContext(this@ExorypadApplication)
            modules(exorypadModule)
        }
    }
}
